lib/stellar_spectrum/services/log.rb

```diff
@@ -0,0 +1,13 @@
+module StellarSpectrum
+  class Log
+
+    def self.write(message)
```

Comment:
We should implement all log levels: `info`, `error`, `warn`, `debug`, etc.

Since they're pretty much the same except the name and the method that is called on the `logger` object, try doing it [dynamically](https://stackoverflow.com/a/3487636/61018).

Reply:
Got it!